doc/adding-new-user-permissions.md

```diff
@@ -0,0 +1,48 @@
+All available permissions are listed in app/policies/permissions.rb. The file also describes a
+set of defualt access roles. These permissions are used in Pundit policies to authorize access 
+and scope models accessible by the user. Each feature requires it's own set of policies, for this
+we use pundits namespaced policy feature (https://github.com/varvet/pundit#policy-namespacing). 
```

Comment:
Since these are all Markdown anyway, could we take advantage of link formatting?

```
we use Pundit's [namespaced policy feature](https://github.com/varvet/pundit#policy-namespacing)
```